src/main/java/ai/elimu/web/content/word/WordCreateController.java

```diff
@@ -272,4 +265,20 @@ private void autoSelectLetterSoundCorrespondences(Word word) {
         
         word.setLetterSoundCorrespondences(letterSoundCorrespondences);
     }
+
+    private void validateWord(Word word, BindingResult result) {
```

Comment:
@tomaszsmy Could you add a unit test for this method, covering some examples of words that are numbers?